### PR TITLE
Improved streamline and structure of AIG providers

### PR DESCRIPTION
**Pull request by daisyfaithauma**
### Summary
-Moved the content from the [partial] file(cloudflare/cloudflare-docs/blob/production/content/ai-gateway/_partials/_anthropic.md) directly to the page and delete the partial file.
- Added headings and front-line description

### CODE REVIEW

File: content/ai-gateway/providers/anthropic.md [hunk 1]
```diff
@@ -4,5 +4,46 @@ pcx_content_type: get-started
...
 
 # Anthropic
+[Anthropic](https://www.anthropic.com/) helps  build reliable, interpretable, and steerable AI systems.
 
-{{<render file="_anthropic.md">}}
\ No newline at end of file
+## Endpoint
+
+`https://gateway.ai.cloudflare.com/v1/{account_id}/{gateway_id}/anthropic`
+
```
Comment (marciocloudflare) [inline content/ai-gateway/providers/anthropic.md:12]:
```suggestion
```

File: content/ai-gateway/providers/azureopenai.md [hunk 2]
```diff
@@ -4,5 +4,6 @@ pcx_content_type: get-started
 ---
 
 # Azure OpenAI
+[Azure OpenAI](https://azure.microsoft.com/en-gb/products/ai-services/openai-service/) allows you apply natural language algorithms on your data.
```
Comment (marciocloudflare) [inline content/ai-gateway/providers/azureopenai.md]:
```suggestion
[Azure OpenAI](https://azure.microsoft.com/en-gb/products/ai-services/openai-service/) allows you to apply natural language algorithms on your data.
```

File: content/ai-gateway/providers/cohere.md [hunk 3]
```diff
@@ -4,5 +4,6 @@ pcx_content_type: get-started
 ---
 
 # Cohere
+[Cohere](https://cohere.com/) build AI models designed to solve real-world business challenges.
```
Comment (marciocloudflare) [inline content/ai-gateway/providers/cohere.md]:
```suggestion
[Cohere](https://cohere.com/) allows you to build AI models designed to solve real-world business challenges.
```

File: content/ai-gateway/providers/openai.md [hunk 4]
```diff
@@ -4,5 +4,6 @@ pcx_content_type: get-started
 ---
 
 # OpenAI
+[OpenAI](https://openai.com/about/)  helps you build with ChatGPT.
```
Comment (marciocloudflare) [inline content/ai-gateway/providers/openai.md]:
```suggestion
[OpenAI](https://openai.com/about/) helps you build with ChatGPT.
```

File: content/ai-gateway/providers/universal.md [hunk 5]
```diff
@@ -5,6 +5,7 @@ weight: 1
 ---
 
 # Universal Endpoint
+You can use the Universal endpoint to contact every provider.
```
Comment (marciocloudflare) [inline content/ai-gateway/providers/universal.md]:
```suggestion
You can use the Universal Endpoint to contact every provider.
```

Comment (marciocloudflare) [reply content/ai-gateway/providers/universal.md]: You had "Universal Endpoint" in the title, but if it's not the name of a feature, please lowercase it in the title as well

**Review by marciocloudflare — APPROVED** [review]

File: content/ai-gateway/providers/universal.md (hunk 5 above)
Comment (daisyfaithauma) [reply content/ai-gateway/providers/universal.md]: Its the name of a feature.

**Pull request merged**